low-level/sspl_ll_d

```diff
@@ -715,6 +717,15 @@ def send_thread_controller_request(state):
 #     logger.info(f"Received SIGUSR1 signal to set log level to '{log_level}'")
 #     logger.setLevel(log_level)
 
+def initialize_iem():
+    try:
+        EventMessage.init(component='sspl', source='S')
```

Comment:
What is source argument?

Reply:
I think it refers to Software as the source or type of IEM being generated

Reply:
> I think it refers to Software as the source or type of IEM being generated

yes